<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>time_label</class>
 <widget class="QWidget" name="time_label">
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="graph">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>861</width>
     <height>651</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="draw">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>351</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Построить график</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>30</y>
     <width>291</width>
     <height>149</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="time_input"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="temp_value"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>* 10 ^ -</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="temp_deg"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>30</y>
     <width>371</width>
     <height>149</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="min_temp">
      <property name="text">
       <string>Мин. температура:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="max_temp">
      <property name="text">
       <string>Макс. температура:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>431</width>
     <height>149</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Время:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="temp_label">
      <property name="text">
       <string>Температуропроводность: </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>1260</x>
     <y>30</y>
     <width>171</width>
     <height>149</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLineEdit" name="min_temp_input"/>
    </item>
    <item>
     <widget class="QLineEdit" name="max_temp_input"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
